docs/release-notes.md

```diff
@@ -33,6 +33,39 @@ n8n uses [semantic versioning](https://semver.org/){:target=_blank .external-lin
 You can find the release notes for older versions of n8n [here](/release-notes/0-x)
 ///
 
+## n8n@1.63.0
+
+View the [commits](https://github.com/n8n-io/n8n/compare/n8n@1.62.3...n8n@1.63.0){:target=_blank .external-link} for this version.<br />
+**Release date:** 2024-10-09
+
+/// warning | [Breaking change](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md){:target=_blank .external-link}
+What changed?
+
+- The worker server used to bind to IPv6 by default. It now binds to IPv4 by default.  
+- The worker server's /healthz used to report healthy status based on database and Redis checks. It now reports healthy status regardless of database and Redis status, and the database and Redis checks are part of /healthz/readiness.  
+
+When is action necessary?
+
+- If you experience a port conflict error when starting a worker server using its default port, set a different port for the worker server with QUEUE_HEALTH_CHECK_PORT.  
+- If you are relying on database and Redis checks for worker health status, switch to checking /healthz/readiness instead of /healthz.  
```

Comment:
```suggestion
- If you are relying on database and Redis checks for worker health status, switch to checking `/healthz/readiness` instead of `/healthz`.  
```